src/games/strategy/sound/ClipPlayer.java

```diff
@@ -442,239 +419,67 @@ private void parseClipPaths(final String pathName, final boolean subFolder) {
     } catch (final Exception e2) {
       thisSoundFile = null;
     }
-    if (thisSoundFile == null || !thisSoundFile.exists()) {
-      // final long startTime = System.currentTimeMillis();
-      // we are probably using zipped sounds. there might be a better way to do this...
-      final String soundFilePath = thisSoundURL.getPath();
-      if (soundFilePath != null && soundFilePath.length() > 5 && soundFilePath.indexOf(".zip!") != -1) {
-        // so the URL with a zip or jar in it, will start with "file:", and unfortunately when you make a file and test
-        // if it exists, if it
-        // starts with that it doesn't exist
-        final int index1 = Math.max(0, Math.min(soundFilePath.length(),
```

Comment:
It looks like you are removing all logic needed to get a sound clip out of a map zip??